docs/misc/downloads-api.mdx

```diff
@@ -0,0 +1,63 @@
+---
+slug: /downloads-api
+description: Paper provides a downloads API that you can use to access builds.
+---
+
+# Downloads API
+
+PaperMC provides a downloads API to facilitate automated downloads access. Full documentation can be
+found on the [Downloads API Docs](https://api.papermc.io/docs).
+
+## Getting Highest Stable Build Number
+
+<VersionFormattedCode language={"shell"}>
+```
+#!/bin/bash
+
+PROJECT="paper"
+MINECRAFT_VERSION="%%_MAJ_MIN_MC_%%"
+
+LATEST_BUILD=$(curl -s https://api.papermc.io/v2/projects/${PROJECT}/versions/${MINECRAFT_VERSION}/builds | \
+    jq -r '.builds | map(select(.channel == "default") | .build) | .[-1]')
+
+echo "Latest build is $LATEST_BUILD"
+```
+</VersionFormattedCode>
+
+This will get the latest stable build for the given project and Minecraft version.
+
+## Downloading Latest Stable Build
+
+<VersionFormattedCode language={"shell"}>
+```
+#!/bin/bash
+
+PROJECT="paper"
+MINECRAFT_VERSION="%%_MAJ_MIN_MC_%%"
+
+LATEST_VERSION=$(curl -s https://api.papermc.io/v2/projects/paper | \
+    jq -r '.versions' | \
+    jq -r '.[-1]')
+
+LATEST_BUILD=$(curl -s https://api.papermc.io/v2/projects/${PROJECT}/versions/${MINECRAFT_VERSION}/builds | \
+    jq -r '.builds | map(select(.channel == "default") | .build) | .[-1]')
+
+JAR_NAME=paper-${LATEST_VERSION}-${LATEST_BUILD}.jar
+
+PAPERMC_URL="https://api.papermc.io/v2/projects/paper/versions/${LATEST_VERSION}/builds/${LATEST_BUILD}/downloads/${JAR_NAME}"
+
+# Download the latest PaperMC version
+wget -O server.jar "$PAPERMC_URL"
+echo "Downloads completed"
+```
+</VersionFormattedCode>
+
+This is the most common use case for the API. It will download the highest stable build for the given project and
+Minecraft version. You should always serve & use the stable builds as experimental builds are prone to error and
```

Comment:
English is not my native language, but I think there's a missing comma before `as experimental builds`:
`You should always serve and use stable builds, as experimental builds are prone to errors and do not receive support.`